extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -26,6 +27,9 @@ const EXPERIMENT = 'amp-sidebar';
 /** @const */
 const TAG = 'AmpSidebar';
 
+/** @const */
+const IOS_SAFARI_BOTTOMBAR_HEIGHT = '10vh';
```

Comment:
This is to make sure we compensate enough for both 6 and 6plus. I custom measured this (not available anywhere in the open) I will make the design doc public after final edits if you think that would be good.